### DISCUSSION TITLE
blog/2023-year-summary

### DISCUSSION THREAD

**Discussion by giscus** [bbruceyuan/bbruceyuan.github.io#5] [Announcements]
# blog/2023-year-summary

2023年，我结婚了，换工作了。这也意味着在 2024 年，一切都是新的开始，所以这篇文章命名为《新的开始》。

https://bruceyuan.com/blog/2023-year-summary.html

**Comment by stevenash0822** [comment]
老哥，境遇太像了，一样是13年高考，一样是23年从鹅厂离开，一样在大模型时代的裹挟中艰难前行，一样没有过多仪式感的和妹子领了证 （主要怪我233）。祝前途顺遂，未来可期~

**Reply by bbruceyuan** [reply]
老哥共勉，希望 24 年更好！～

**Reply by liuyang4194** [reply]
共勉

**Comment by wangdefou-dev** [comment]
新年快乐~
我也是23年和小姐姐扯得证儿
搞了一场仪式感还比较强的婚礼，哭得稀里哗啦的那种
希望24年老哥快乐幸福呀~

**Reply by bbruceyuan** [reply]
你婚礼听着就很幸福👍～（哭得稀里哗啦）

24 年冲啊

**Comment by DayuGuo** [comment]
恭喜！
